src/ts/greedy-nav/greed-nav.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Core styles for Cads GreedyNav
```

Comment:
Shouldn't this live within the scss dir?

Reply:
SCSS I co-located to group by function, but I can move it. no biggie. you think I should right?